nemo/collections/llm/gpt/model/llama.py

```diff
@@ -949,6 +960,30 @@ def config(self) -> "HFLlamaConfig":
         """
         source: LlamaConfig = io.load_context(str(self), subpath="model.config")
 
+        if type(source).__name__ == "Llama33NemotronSuper49BConfig":
+            assert hasattr(source, 'heterogeneous_layers_config_encoded_json')
```

Comment:
Can we change this to model_name / model_path instead like we do in llama_nemotron ? 

We aren't guaranteed to have the HF checkpoint available in the cache and this will attempt to download the config again.